Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py

```diff
@@ -0,0 +1,1150 @@
+"""Varonis Data Security Platform integration
+"""
+
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any, List, Tuple
+from requests_ntlm import HttpNtlmAuth
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CONSTANTS '''
+
+MAX_INCIDENTS_TO_FETCH_PER_RUN = 5000
+MAX_INCIDENTS_TO_FETCH = 200
```

Comment:
MAX_INCIDENTS_TO_FETCH - is max number of alerts that we would request from api
MAX_INCIDENTS_TO_FETCH_PER_RUN - max number of incidents we would try to retrieve during one run of fetching incidents. For example, it is first run of fetching incidents and you are fetching incidents for last 2 weeks and there is 6k of incidents. So during first run, it would fetch only 5k of incidents